libs/langchain-community/src/agents/toolkits/connery/index.ts

```diff
@@ -28,8 +23,7 @@ export class ConneryToolkit extends Toolkit {
     toolkit.tools = [];
 
     const actions = await conneryService.listActions();
-    toolkit.tools.push(...actions);
-
+    toolkit.tools.push(...actions.map((action) => action as unknown as Tool)); // This is a hack to make TypeScript happy, as TypeScript doesn't know that ConneryAction (StructuredTool) extends Tool.
```

Comment:
Hey @jacoblee93, I just wanted to bring this up. I haven't found a better way to handle the TypeScipt error than asserting the `StructuredTool` to the `Tool` through `unknown`. Is there any other way to avoid such a hack?

Reply:
Yeah, we need to allow `Toolkit` to allow `StructuredTool`s. This is fine for now :(